metron-interface/metron-alerts/e2e/login/login.e2e-spec.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { LoginPage } from './login.po';
+
+describe('login to application', function() {
+    let page: LoginPage;
+
+    beforeEach(() => {
+        page = new LoginPage();
+    });
+
+    it('should display error message for invalid credentials', () => {
```

Comment:
@iraghumitra I have noticed that if I have any other builds running at the same time as running the E2E tests, I get failures for the "should delete first search items from search box having multiple search fields" test.  If I run them alone, then they all pass.  That's good enough, but just thought I would mention it in case others run into the same problem.